pkg/collector/reconcile/service_test.go

```diff
@@ -130,6 +130,24 @@ func TestDesiredService(t *testing.T) {
 
 	})
 
+	t.Run("should return service with local internal traffic policy", func(t *testing.T) {
+
+		grpc := "grpc"
+		jaegerPorts := v1.ServicePort{
+			Name:        "jaeger-grpc",
+			Protocol:    "TCP",
+			Port:        14250,
+			AppProtocol: &grpc,
+		}
+		p := paramsWithMode(v1alpha1.ModeDaemonSet)
+		ports := append(p.Instance.Spec.Ports, jaegerPorts)
+		expected := serviceWithInternalTrafficPolicy("test-collector", ports, v1.ServiceInternalTrafficPolicyLocal)
+		actual := desiredService(context.Background(), p)
+
+		assert.Equal(t, expected, *actual)
+
```

Comment:
please remove the redundant empty lines in the test

Reply:
done